src/functions/genfrac.js

```diff
@@ -321,3 +321,68 @@ defineFunction({
         }, parser.mode);
     },
 });
+
+// $FlowFixMe
+const delimFromValue = function(delimString: string): string | null {
+    let delim = null;
+    if (delimString.length > 0) {
+        delim = delimString;
+        delim = delim === "." ? null : delim;
+    }
+    return delim;
+};
+
+// Infix fractions with more operands than just numerator and denominator
+
+defineFunction({
+    type: "infix",
+    names: ["\\atopwithdelims", "\\overwithdelims"],
+    // These two functions allow the author to choose the delims, but not
+    // the bar thickness.
+    props: {
+        numArgs: 2,
+        infix: true,
+    },
+    handler({parser, funcName, token}, args) {
+        const [leftDelim, rightDelim] = args;
+        return new ParseNode("infix", {
+            type: "infix",
+            replaceWith: "\\" + funcName + "frac",
+            leftDelim: leftDelim,
+            rightDelim: rightDelim,
+            token: token,
+        }, parser.mode);
+    },
+});
+
+defineFunction({
+    type: "genfrac",
+    names: ["\\\\atopwithdelimsfrac", "\\\\overwithdelimsfrac"],
+    props: {
+        numArgs: 4,
+    },
+    handler: ({parser, funcName}, args) => {
+        const [numer, infixNode, denom] = args;
+
+        // Look into the parse node to get the desired delimiter.
+        // $FlowFixMe
+        const leftDelim = delimFromValue(infixNode.value.leftDelim.value);
```

Comment:
There are a couple of issues with the typing:
- `leftDelim` may not be defined
- if it defined, its type (as defined in ParseNode.js) is `ParseNode` which makes the type of `leftDelim.value` `NodeValue` which could be a lot of things (at least the way things are typed right now).

Reply:
I was able to figure out how to fix the `$FlowFixMe`s.  Here's the updated `handler`:
```
    handler: (context, args) => {
        const [numer, , denom] = args;

        const infixNode = assertNodeType(args[1], "infix");

        // Look into the parse node to get the desired delimiter.
        const leftDelim = infixNode.value.leftDelim
            ? delimFromValue(
                checkDelimiter(infixNode.value.leftDelim, context).value)
            : null;
        const rightDelim = infixNode.value.rightDelim
            ? delimFromValue(
                checkDelimiter(infixNode.value.rightDelim, context).value)
            : null;
        const hasBarLine = context.funcName === "\\\\overwithdelimsfrac";

        return new ParseNode("genfrac", {
            type: "genfrac",
            numer: numer,
            denom: denom,
            continued: false,
            hasBarLine: hasBarLine,
            leftDelim: leftDelim,
            rightDelim: rightDelim,
            size: "auto",
        }, context.parser.mode);
    },
```

Reply:
I've also exported `checkDelimiter` from delimsizing.js and imported it into this file.